databricks/koalas/tests/indexes/test_datetime.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright (C) 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from distutils.version import LooseVersion
+
+import pandas as pd
+
+import databricks.koalas as ks
+
```

Comment:
nit: don't need this empty line?

Reply:
Removed, thanks!

Reply:
Oops, sorry. I misread the change.

I meant the empty line between `import databricks.koalas as ks` and `from databricks.koalas ...`.
We need an empty line between `import pandas ...` and `import databricks.koalas ...`.

@xinrong-databricks Could you address in your next PR?

Reply:
Sounds good!